Sources/MapboxMaps/Viewport/States/Overview/OverviewViewportStateOptions.swift

```diff
@@ -0,0 +1,22 @@
+import Turf
+import CoreLocation
+
+public struct OverviewViewportStateOptions: Equatable {
+    public var geometry: Geometry
+    public var padding: UIEdgeInsets
+    public var bearing: CLLocationDirection
+    public var pitch: CGFloat
+    public var animationDuration: TimeInterval
```

Comment:
@pengdev I didn't use the word "frame" in this option name. Same in `FollowingViewportStateOptions`. We can reevaluate the names for RC.

Reply:
sure